modules/pxp-module-puppet

```diff
@@ -235,13 +286,11 @@ if __FILE__ == $0
   if action == 'metadata'
     puts metadata.to_json
   else
-    params_and_config = JSON.parse($stdin.read.chomp) rescue nil
+    args = JSON.parse($stdin.read.chomp) rescue nil
 
-    run_result = run(params_and_config)
-    puts run_result.to_json
+    action_results = action_run(args)
+    print action_results.to_json
 
-    if !run_result["error"].nil?
-      exit 1
-    end
+    exit 1 unless action_results["error"].nil?
```

Comment:
Preferred the original expression of this (explicit if vs adverbial unless).  Adverbial unless shifts the eye to the action, but I think when following this code the test is more important to encounter first.